tests/syft/lib/allowlist_test.json

```diff
@@ -720,6 +720,13 @@
           { "data_types": ["float16"], "lte_version": "1.5.1" }
         ]
       },
+      "angle": {
+        "profile": "tensor_method_noinput",
+        "not_available": [
+          { "data_types": ["bool"] },
```

Comment:
Add an "lte_version": "1.6.0" to the bool rule so that it gets automatically checked in 1.7.0 later.
I think its a good idea we never add blanket rules without version limits so we don't forget to later test these if new versions are released.